.pipelines/rp-int-deploy.yml

```diff
@@ -0,0 +1,88 @@
+trigger: none
```

Comment:
Should we rename this file to something like `rp-full-dev-setup.yaml` to avoid confusion? Otherwise, it sounds like we are deploying INT, for which there are pipelines in the ADO repo.

If it that makes sense, I would avoid references to int for that same reason in the file content. Same for the name of the rp-int-deployment directory under templates.

Maybe we should rename too the files rp-params.yml and rp-vars.yml since there are other pipelines in this folder deploying RPs but they won't be using these files.

Reply:
Agreed, we can use "int-like" if we have to, but we usually just refer to it as a full RP environment so rp-full-dev-setup.yaml makes sense.